src/test/CMakeLists.txt

```diff
@@ -1,83 +1,19 @@
-# Copyright 1998-2019 Lawrence Livermore National Security, LLC and other
```

Comment:
Why was all of this removed (e.g., the license header, the sequential make option, etc.)?

Reply:
The license header getting removed was my mistake. Thanks for catching it! 

As for the remaining changes, I've changed the build structure so that it no longer builds a separate project for the tests, it will now build them in hypre_BINARY_DIR/test. My reasoning was to avoid any issues with using one set of configurations to build hypre, then a different set to build hypre-test. 

If creating a separate location hypre-test (not tied to the hypre_BINARY_DIR) is desired, I can keep the original hypre-test structure and pull the same configuration options that were used for hypre.

Reply:
This makes perfect sense.  One question, though.  Are the test drivers always built or is an additional "target" needed to make them?  Users who are only interested in building/installing the library may not want to worry about the test drivers.

Thanks!

Reply:
It's a cmake option that specifies whether or not to build the test drivers (similarly for the example drivers), then as you mentioned users can specify which drivers (if any) are built. 

Both options (HYPRE_BUILD_TESTS and HYPRE_BUILD_EXAMPLES) are off by default.

Reply:
Works for me.  Thanks!